webpack/ForemanTasks/Components/TasksTable/TasksTableSelectors.js

```diff
@@ -0,0 +1,44 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import { selectForemanTasks } from '../../ForemanTasksSelectors';
+
+export const selectTasksTable = state =>
+  selectForemanTasks(state).tasksTable || {};
+
+export const selectTasksTableContent = state =>
+  selectTasksTable(state).tasksTableContent || {};
+
+export const selectTasksTableQuery = state =>
+  selectTasksTable(state).tasksTableQuery || {};
+
+export const selectQuery = state => selectTasksTableQuery(state).query || {};
+
+export const selectLoading = state =>
+  selectTasksTableQuery(state).loading || false;
+
+export const selectPagitation = state =>
+  selectTasksTableQuery(state).pagination || {};
+
+export const selectItemCount = state =>
+  selectTasksTableQuery(state).itemCount || 0;
+
+const formatDate = date =>
```

Comment:
I don't think it belongs here. In the component level, you can import a date component from `foremanReact` who does the same.